examples/process-persistence/README.md

```diff
@@ -0,0 +1,373 @@
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+-->
+
+# Example :: Process Persistence
+
+This example depicts the usage of persistence configuration for various supported databases like H2, Postgresql and
```

Comment:
Please remove `supported` from this sentence. The list of supported databases is tracked separately, and H2 is not among them. H2 is provided for OOTB developer experience only.